src/Compiler/CompilerUtils.cpp

```diff
@@ -40,13 +40,16 @@ using namespace mlir;
 using namespace onnx_mlir;
 
 const std::string OnnxMlirEnvOptionName = "ONNX_MLIR_FLAGS";
-#if defined(ONNX_MLIR_REPOSITORY) && defined(ONNX_MLIR_REVISION) &&            \
+static const std::string OnnxMlirVersion = "onnx-mlir version 1.0.0";
```

Comment:
I think at this point it's probably worth moving all of this to a function call similar to how clang does it. Then when we move to having a version file (which we probably really should), it would also make things much simpler.